src/main/webapp/app/exercises/file-upload/stage/file-upload-stage.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, ElementRef, EventEmitter, Input, Output, ViewChild } from '@angular/core';
+import { MAX_SUBMISSION_FILE_SIZE } from 'app/shared/constants/input.constants';
+import { FileDetails } from 'app/entities/file-details.model';
+import { FileUploadSubmission } from 'app/entities/file-upload-submission.model';
+import { AlertService } from 'app/core/util/alert.service';
+import { FileService } from 'app/shared/http/file.service';
+
+@Component({
+    selector: 'jhi-stage',
```

Comment:
```suggestion
    selector: 'jhi-file-upload-stage',
```
A more specific name would be useful to understand what this component is in places where it is used without having to look at further details.